lib/plucky/normalizers/sort_value.rb

```diff
@@ -18,11 +18,7 @@ def initialize(args = {})
       def call(value)
         case value
           when Array
```

Comment:
This is now a `case` statement with a single comparison check and an else. Mine as well make it `if`.  Or a short circuit that's just `return blah if blah; default_case`